pkg/services/dashboardsnapshots/service/service_test.go

```diff
@@ -96,66 +80,3 @@ func TestIntegrationDashboardSnapshotsService(t *testing.T) {
 		require.Equal(t, rawDashboard, decrypted)
 	})
 }
-
-func TestIntegrationValidateDashboardExists(t *testing.T) {
```

Comment:
This test was only testing [this](https://github.com/grafana/grafana/blob/219672722662938633ffae3f6e8bfa89f8f29661/pkg/services/dashboardsnapshots/service/service.go#L32). Moved it up a level to test it being used [here](https://github.com/grafana/grafana/blob/219672722662938633ffae3f6e8bfa89f8f29661/pkg/services/dashboardsnapshots/service.go#L52) -- see pkg/services/dashboardsnapshots/service_test.go